client/state/media/actions/utils.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import Dispatcher from 'dispatcher';
+
+/**
+ * Constants
+ */
+const ONE_YEAR_IN_MILLISECONDS = 31540000000;
+
+export const getBaseTime = () => Date.now() + ONE_YEAR_IN_MILLISECONDS;
+export const getTransientDate = ( baseTime = getBaseTime(), index = 0, fileCount = 1 ) =>
+	new Date( baseTime - ( fileCount - index ) ).toISOString();
+
+export const dispatchFluxCreateMediaItem = ( transientMedia, site ) =>
+	Dispatcher.handleViewAction( {
+		type: 'CREATE_MEDIA_ITEM',
+		siteId: site.ID,
+		data: transientMedia,
+		site,
+	} );
+
+export const dispatchFluxReceiveMediaItemSuccess = ( transientMediaId, siteId, savedMediaItem ) =>
+	Dispatcher.handleServerAction( {
+		type: 'RECEIVE_MEDIA_ITEM',
```

Comment:
We can adjust it going forward, or even create separate functions for each of those cases. Do you have a preference based on what would be easiest to clean up in the future PR when we strip out flux entirely? It may be easier to identify what we're removing at each step if we have a different function, but I'm not totally sure if that's necessary.

Reply:
> We can adjust it going forward, or even create separate functions for each of those cases.

You're right 👍  We're going to remove these anyways sometime soon